### DISCUSSION TITLE
动态过滤 能否增加过滤充电动态的功能？

### DISCUSSION THREAD

**Discussion by KuiPer-LTW** [the1812/Bilibili-Evolved#4991] [功能建议 / Ideas]
![image](https://github.com/user-attachments/assets/fe4025ec-1a83-4051-b7cb-938ede813b17)

**Comment by Geeyun-JY3** [comment]
请善用搜索：[`动态过滤器 - 移除充电专属动态`](https://github.com/the1812/Bilibili-Evolved/blob/dbd813b10041aeec0a4e1867be4a4f75c835bd5b/doc/features/features.md#%E5%8A%A8%E6%80%81%E8%BF%87%E6%BB%A4%E5%99%A8---%E7%A7%BB%E9%99%A4%E5%85%85%E7%94%B5%E4%B8%93%E5%B1%9E%E5%8A%A8%E6%80%81)

**Reply by KuiPer-LTW** [reply]
谢谢
